test.sh.in

```diff
@@ -358,6 +292,111 @@ fetchOVALog(){
     popd
 }
 
+dockerUp(){
+    pushd $WORKSPACE
+    echo $SUDO_PASSWORD |sudo -S docker load -i rackhd_pipeline_docker.tar
+    popd
+    cp -r ${WORKSPACE}/build-deps ${WORKSPACE}/build-config/jobs/pr_gate/docker
+    pushd ${WORKSPACE}/build-config/jobs/pr_gate/docker
+    #cp -r ${WORKSPACE}/build-config/jobs/pr_gate/docker/* .
+    echo $SUDO_PASSWORD |sudo -S docker build -t my/test .
+    echo $SUDO_PASSWORD |sudo -S docker run --net=host -v /etc/localtime:/etc/localtime:ro -d -t my/test
+    popd
+}
+
+setupTestsConfig(){
+    if [ "$RUN_FIT_TEST" == true ] ; then
+        pushd ${WORKSPACE}/build-config/jobs/pr_gate/test_config/FIT
+        sed -i "s/RACKHD_SSH_USER = vagrant/RACKHD_SSH_USER = ${SUDO_USER}/g" config.ini
+        sed -i "s/RACKHD_SSH_PASSWORD = vagrant/RACKHD_SSH_PASSWORD = $SUDO_PASSWORD/g" config.ini
+        sed -i "s/\"username\": \"vagrant\"/\"username\": \"${SUDO_USER}\"/g" credentials_default.json
+        sed -i "s/\"password\": \"vagrant\"/\"password\": \"$SUDO_PASSWORD\"/g" credentials_default.json
+        popd
+        cp ${WORKSPACE}/build-config/jobs/pr_gate/test_config/FIT/*.* ${WORKSPACE}/RackHD/test/config
```

Comment:
Thanks.
Fixed with PR[](https://github.com/RackHD/RackHD/pull/802) to add a new stack to rackhd/test/config/stack_config.json